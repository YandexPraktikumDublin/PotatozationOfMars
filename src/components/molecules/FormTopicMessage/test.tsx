import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { FormTopicMessage } from '.'

describe('<FormTopicMessage />', () => {
  const body = 'Test body'
  const userName = 'Ivan Ivanov'
  const date = '12.02.20'

  it('should renders correct <FormTopicMessage />', () => {
    const wrapper = shallow(
      <FormTopicMessage body={body} userName={userName} date={date} />
    )

    expect(wrapper.contains(body)).toBeTruthy()
    expect(wrapper.contains(userName)).toBeTruthy()
    expect(wrapper.contains(date)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
