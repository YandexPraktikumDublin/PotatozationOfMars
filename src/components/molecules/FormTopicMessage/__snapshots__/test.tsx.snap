// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormTopicMessage /> should renders correct <FormTopicMessage /> 1`] = `
<div
  className="text-left border border-primary rounded-2xl p-4 mb-4 last:mb-0 dark:border-white"
>
  <div
    className="mb-4"
  >
    Test body
  </div>
  <div
    className="text-xs"
  >
    Ivan Ivanov
     
    12.02.20
  </div>
</div>
`;
